core/services/keeper/orm.go

```diff
@@ -106,36 +107,47 @@ DELETE FROM upkeep_registrations WHERE registry_id IN (
 	return rowsAffected, nil
 }
 
-//EligibleUpkeepsForRegistry fetches eligible upkeeps for processing
-//The query checks the following conditions
-// - checks the registry address is correct and the registry has some keepers associated
-// -- is it my turn AND my keeper was not the last perform for this upkeep OR my keeper was the last before BUT it is past the grace period
-// -- OR is it my buddy's turn AND they were the last keeper to do the perform for this upkeep
+/*
```

Comment:
wasn't able to comment on unchanged lines, there are more functions that need changing: BatchDeleteUpkeepsForJob

Reply:
updated!